package.json

```diff
@@ -77,7 +77,7 @@
     "jquery.easing": "^1.4.1",
     "jquery.transit": "^0.9.12",
     "katex": "^0.10.2",
-    "lodash": "^4.17.11",
+    "lodash": "^4.17.15",
```

Comment:
Why is only `loadsh` updated in `package.json`, but a lot of packages get updated in `package-lock.json`?

Reply:
> Why is only `loadsh` updated in `package.json`, but a lot of packages get updated in `package-lock.json`?

these are the dependencies needed by lodash as they update it.

Reply:
Looks like you also upgraded `babel` in lock file.